hack/samples/vllm-samples.md

```diff
@@ -2,8 +2,8 @@
 
 
 Notes: 
-1. The following describes setting up vllm deployment and a guidellm job on OpenShift cluster.
-2. We use  `vllm-test` namespace for vllm deployment components and the load generator jobs. If the namespace doesn't already exists, create one by running `oc create ns vllm-test`.
+1. The following describes setting up vllm deployment on OpenShift cluster.
+2. All vLLM components will run in the `vllm-test` namespace. If the namespace doesn't already exists, create one by running `oc create ns vllm-test`.
```

Comment:
Can you add a note here that these instructions do not consume llmd infra components, please?